gtk/Application.cc

```diff
@@ -1357,6 +1357,10 @@ void Application::Impl::on_prefs_changed(tr_quark const key)
         tr_sessionSetDeleteSource(tr, gtr_pref_flag_get(key));
         break;
 
+	case TR_KEY_torrent_complete_verify_enabled:
+		tr_sessionSetCompleteVerifyEnabled(tr, gtr_pref_flag_get(key));
+		break;
```

Comment:
This has no effect until we add an actual UI element for this pref. In any case this does no harm, so it can stay.

Reply:
I'll fix the spacing/tabulation later Today.

Reply:
4fef9ae956859e5b347260245ca5df3a6aac08c8